src/napari_topostats/_widget_function.py

```diff
@@ -13,6 +13,7 @@
 from napari.viewer import Viewer
 from pandas import DataFrame
 from qtpy.QtWidgets import (
+    QCheckBox,  # Added QCheckBox import
```

Comment:
```suggestion
    QCheckBox,
```

Will be evident from the commit.